selendroid-provider/src/main/java/com/paypal/selion/selendroid/platform/grid/SeLionSelendroidDriver.java

```diff
@@ -344,4 +339,15 @@ private void performLongClickAction(Point point) {
     private void performSwipeAction(Point start, Point end) {
         new TouchActions(this).down(start.getX(), start.getY()).move(end.getX(), end.getY()).up(end.getX(), end.getY()).perform();
     }
+
+    @Override
+    public void rotate(DeviceRotation deviceRotation) {
+        //TODO
```

Comment:
SelendroidDriver implements Rotatable interface which added these two new methods in sel3.  We'll have t o either implement them here or have the next version of selendroiddriver (> 0.17) implements them when it supports sel3.